class.jetpack.php

```diff
@@ -310,6 +310,10 @@ public static function init() {
 	 * Must never be called statically
 	 */
 	function plugin_upgrade() {
+		if( isset( $_GET['sync'] ) ) {
+			error_log('sync do action called ' );
```

Comment:
You'll probably want to remove that `error_log()`. I assume it was there for debugging?